autotest/gcore/misc.py

```diff
@@ -292,7 +292,14 @@ def misc_6_internal(datatype, nBands, setDriversDone):
                     or datatype == gdal.GDT_UInt16
                 ):
                     skip = True
-
+            # FIXME: this shouldn't be happening
```

Comment:
is it still needed? This would indicate that there's some bug in the driver that should ideally be fixed (the above case for JPEG2000 is obsolete since the JPEG2000 driver has been removed for long)